src/providers/analytics/provider.ts

```diff
@@ -58,7 +58,10 @@ export class IndexAnalyticsProvider implements AnalyticsProvider {
       include24hrVolume: true,
     },
   ): Promise<IndexAnalytics> {
-    const token = getIndexTokenDataByAddress(address, chainId)
+    const indices = getChainTokenList(chainId, ["index"])
+    const token = indices.find((token) =>
```

Comment:
also here if you need an individual token, just do

```typescript
const token = getTokenByChainIdAndAddress(chainId, address)
```

Reply:
We want to make sure to only have our indices here

Reply:
ah i see okay